.circleci/config.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       - run:
           name: Run Tests
           command: |
+            set -e
```

Comment:
this is the default behavior?  https://circleci.com/docs/2.0/configuration-reference/#default-shell-options

Reply:
yeah, i added to see if it would fix the ci issue, but didn't work